coala_quickstart/interaction/Logo.py

```diff
@@ -0,0 +1,60 @@
+from coala_quickstart.Strings import coala_bear_logo, welcome_messages
+
+
+def text_wrap(*args, delimiter=' ', end='\n', limit=80):
+    """
+    Breaks it word wise when going further than ``limit`` chars.
+
+    :param args:
+        The stuff you want to have printed.
+    :param delimiter:
+        Will be placed between all the args.
+    :param limit:
+        Char limit.
+    :param end:
+        An ending, will be put as last character and doesn't count into the
+        char limit.
+    :return:
+        A list of strings, each element to be written on its own line.
+    """
+    output = delimiter.join(args)
+    lines = output.splitlines(keepends=True)
+    results = [""]
+    for line in lines:
+        curr_print = line
+        while len(curr_print.rstrip('\n')) > limit:
+            splitpos = curr_print[:limit].rfind(' ')
+            if splitpos < 0:
+                # Word too long, search for a space left from limit at least
+                splitpos = curr_print.find(' ')
+                if splitpos < 0:
+                    break  # Break out and add the long thing in the next line
+
+            results.append(curr_print[:splitpos])
+            curr_print = curr_print[splitpos+1:]
+
+        results.append(curr_print)
+
+    return results
+
+
+def print_welcome_message(printer):
```

Comment:
Me too! Though I'd make one function that prints two given strings side by side generically.

And add a doctest for that ones with some minimal examples.